package/samplers/value_at_risk/sampler.py

```diff
@@ -597,7 +597,23 @@ def get_robust_trial(
         )
         gpr = self._get_gpr_list(study, search_space)[0]
         internal_search_space = gp_search_space.SearchSpace(search_space)
-        X_train = internal_search_space.get_normalized_params(trials)
+        if self._nominal_ranges:
```

Comment:
I think it's better to branch using `self._noisy_suggestion` instead of `self._nominal_ranges`. Or simply always using nominal parameters may be better.